config/settings.py

```diff
@@ -20,62 +22,67 @@
 # See https://docs.djangoproject.com/en/5.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-d)s!h6%_@!q3elll0y5nw)937kfk)@k(@l%m1yn7byi&8qss^l'
+SECRET_KEY = os.getenv("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS: List[str] = []
```

Comment:
As of Python 3.9, type aliases like List, Tuple, Dict, ... are [deprecated](https://docs.python.org/3/library/typing.html#deprecated-aliases).
So, from then on, use the built-in types list, tuple, dict, ...

Use: list[str] instead